src/Runner.Worker/JobExtension.cs

```diff
@@ -141,6 +141,10 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         context.Output($"Fail to parse and display GITHUB_TOKEN permissions list: {ex.Message}");
                         Trace.Error(ex);
                     }
+                    
+                    var secretSource = context.GetGitHubContext("secret_source");
+                    ArgUtil.NotNull(secretSource, nameof(secretSource));
+                    context.Output($"Secret source: {secretSource}");
```

Comment:
Should we add some doc links or context to explain this a little bit? 
I would have no idea about what `Secret source: Actions` means.